tests/e2e-ui-tests/utils/constants.js

```diff
@@ -0,0 +1,130 @@
+module.exports = {
+  auth: {
+    DEFAULT_PASSWORD: '123',
+    INVALID_PASSWORD: 'xxx',
+    NEW_PASSWORD: '12345',
+    usernames: {
+      TREVER_SMITH: 'trever',
+      DAISY_DIALER: 'daisy',
+      MAX_A_MILLION: 'max',
+      PHYLIS_LEXY: 'phylis',
+      JOHNNY_FIZAL: 'johnny',
+      BILLY_BALLER: 'billy',
+      SALLY_SHORER: 'sally',
+      DANNY_DOORMAN: 'danny',
+      INVALID_USER: 'invalid'
+    },
+    userFullName: {
+      TREVER: 'Trever Smith',
+      DAISY: 'Daisy Dialer'
+    }
+  },
+  HASH: '#',
+  login: {
+    URL: '/login',
+    DEFAULT_HEADING: 'FeedHenry WFM Demo',
+    DEFAULT_BODY: 'Trouble logging in? Contact the switchboard.',
+    AUTH_FAIL_MSG: 'Authentication Failed! Try Again.',
+    VALUE_ATTRIBUTE: 'value',
+    USERNAME_MISSING_MSG: 'A username is required.',
+    PASSWORD_MISSING_MSG: 'A password is required.'
+  },
+  logout: {
+    URL: '/login',
+    DEFAULT_BODY: 'Trouble logging in? Contact the switchboard.'
+  },
+  schedule: {
```

Comment:
schedule, groups, workers, files etc. have all been removed from the applications